src/content/ecosystem-explorer/projects/themis-protocol.md

```diff
@@ -12,7 +12,7 @@ tech:
 image:
   src: /assets/images/ThemisProtocol.png
   alt: Themis Logo
-website: https://themis.capital
+website:
```

Comment:
@matthendrian - Is this project still alive? The site seems to be dead. If not - What should we do? Remove it?

Reply:
It looks somewhat active on Twitter, but its websites are all removed or not functional. Would it be possible to move this to some kind of hidden archive for now, then reactivate it later if/when we get updated website links?

 If not, let's just remove it and re-submit it again if anything changes in the future.